### PR TITLE
fix(e2e): add scroll bounded test [MRF-83559]

### PR DESCRIPTION
**Pull request by oscard0m**

### CODE REVIEW

File: experiences/homepage/homepage.test.js [hunk 1]
```diff
@@ -62,7 +62,42 @@ const homepageTest = function() {
 		expect(rightContentLoaded).equal(true);
 	});
 
+	it('scroll bounded', async function() {
+		let isAtInitialSnapPoint = await isAtSnapPoint(
+			browser,
+			config.cards.homepage.cardSelector,
+			623
```
Comment (oscard0m) [inline experiences/homepage/homepage.test.js]: TODO: implement utility to get currentSnapPoint

File: experiences/homepage/homepage.test.js [hunk 2]
```diff
@@ -52,6 +52,54 @@ const homepageTest = function() {
 		expect(cardExists).equal(true);
 	});
 
+	it('scroll bounded', async function() {
+		//TODO implement utility to get current SnapPoint
+		let currentSnapPoint = 623;
```
Comment (oscard0m) [inline experiences/homepage/homepage.test.js]: Applying this change:
image

image


I get different values:
image

Any clue why is that if the same code is executed (supposedly) and no scrolls in between are happening (supposedly)?

@davidmohringmrf

File: experiences/homepage/homepage.test.js [hunk 3]
```diff
@@ -52,6 +52,54 @@ const homepageTest = function() {
...
+		// First 60px transitioner doesn't made the card appear
+		const pixelsWithoutAnimation = 60;
+		const smallScrollPixels = 20;
+		// Transform to pixels
+		// TODO to calculate relative factor
+		const scrollBoundFactor = 2.1;
```
Comment (oscard0m) [inline experiences/homepage/homepage.test.js]: Not sure which is the formula to get this factor. Pending to test it with different screen sizes too. Any theory on how to calculate this? @davidmohringmrf

File: experiences/homepage/homepage.test.js [hunk 4]
```diff
@@ -52,8 +51,54 @@ const homepageTest = function() {
...
 	});
 
+	it('scroll bounded', async function() {
+		// Initial 300px scroll to trigger
+		// First 60px transitioner doesn't made the card appear
+		const initialBrowserScroll = 300 + 60;
+		const smallScrollPixels = 20;
+		const cardSelector = config.cards.homepage.cardSelector;
+		const scrollBoundedSnapPoints = [];
+		const timeout = 150;
```
Comment (oscard0m) [inline experiences/homepage/homepage.test.js:61]: with the default `timeout` of `100` tests were not working (different values when putting breakpoints or not).

**Review by davidmohringmrf — APPROVED** [review]

**Pull request merged**